### PR TITLE
[ALLUXIO-1865] [Alluxio-Perf] Update perf branch

### PR DESCRIPTION
**Pull request by RongGu**
https://alluxio.atlassian.net/browse/ALLUXIO-1865

The perf branch has been merged with 1.0.2-SNAPSHOT. This PR updates the implementation.

### CODE REVIEW

File: perf/pom.xml [hunk 1]
```diff
@@ -27,42 +27,41 @@
...
       <groupId>org.alluxio</groupId>
       <artifactId>alluxio-core-common</artifactId>
-      <version>${alluxio.version}</version>
+      <version>${project.version}</version>
     </dependency>
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-client</artifactId>
-      <version>${alluxio.version}</version>
+      <artifactId>alluxio-core-client-internal</artifactId>
```
Comment (yupeng9) [inline perf/pom.xml]: `alluxio-core-client-internal` is removed in 1.0 branch?

File: perf/src/main/java/alluxio/perf/fs/AlluxioPerfFS.java [hunk 2]
```diff
@@ -62,7 +62,6 @@ private AlluxioPerfFS() {
    *
    * @throws IOException
    */
-  @Override
```
Comment (yupeng9) [inline perf/src/main/java/alluxio/perf/fs/AlluxioPerfFS.java]: The annotation can be retained?

**Comment by yupeng9** [comment]
Thanks for the perf update! I left some minor comments.

File: perf/pom.xml (hunk 1 above)
Comment (RongGu) [inline perf/pom.xml]: Currently, the perf branch still has `alluxio-core-client-internal`.

File: perf/src/main/java/alluxio/perf/fs/AlluxioPerfFS.java (hunk 2 above)
Comment (RongGu) [inline perf/src/main/java/alluxio/perf/fs/AlluxioPerfFS.java]: Oh, it's an incorrect setting in my Intellij IDE that makes the annotation be retained.
I'll add them back.

File: perf/pom.xml (hunk 1 above)
Comment (yupeng9) [inline perf/pom.xml]: ah, I see. `internal` is still in branch-1.0 branch.

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**